src/components/LoanCards/GridLoanCard.vue

```diff
@@ -250,12 +260,15 @@ export default {
 			padding-left: 0;
 		}
 	}
+}
 
-	.grid-loan-card {
-		width: rem-calc(280);
-		@include breakpoint(340px down) {
-			min-width: rem-calc(256);
-			width: rem-calc(256);
+.is-in-featured {
+	&.column-block {
+		padding: 0 rem-calc(10);
+		margin-bottom: 2.5rem;
```

Comment:
this was the actual fix. just had to move stuff around in order to only apply that to the is-in-featured class in a way the lint rules allowed.